test_schemas/en/test_skipping_to_questionnaire_end_multiple_sections.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
Isn't this routing to end rather than skipping? I couldn't see _skip_conditions_

Reply:
Schemas for skip conditions are prefixed with `skip_conditions`. I used `skipping` as that is what the existing schema was called for `skipping` to section end. I started renaming this to routing, but then it didn't sound right. It is about when we `skip`/`jump` to a questionnaire end that just route to it even if the underlying use is `routing rules`. 

Open to other suggestions.

Reply:
I think it is clearer to name it after the behaviour than the technical details, the tests etc just didn't read quite right to me when it said routing, maybe that's just me. I'm not too fussed so have updated all references to `routing` instead of `skipping` for this and the existing schema.